website/src/documentation/programming-guide.md

```diff
@@ -1606,17 +1630,34 @@ class ProcessRecord(beam.DoFn):
      pass  
   
 ```
+
 {:.language-java}
 **PaneInfo:**
 When triggers are used, Beam provides a `PaneInfo` object that contains information about the current firing. Using `PaneInfo`
 you can determine whether this is an early or a late firing, and how many times this window has already fired for this key.
 
+{:.language-py}
+**PaneInfo:**
```

Comment:
Let's add a link to https://issues.apache.org/jira/browse/BEAM-3759

I do not believe this currently works in Python.

Reply:
Added the link.